__tests__/core/FlureeClient.test.ts

```diff
@@ -836,6 +820,132 @@ describe('FlureeClient', () => {
+      });
+
+      const did = client.getDid();
+
+      if (!did) {
+        fail('DID not defined');
+      }
+
+      await client
+        .transact({
+          '@context': {
+            'f:equals': { '@container': '@list' },
+          },
+          insert: [
+            {
+              '@id': 'ex:freddy',
+              '@type': 'ex:Yeti',
+              'ex:yetiSecret': "freddy's secret",
+            },
+            {
+              '@id': 'ex:letty',
+              '@type': 'ex:Yeti',
+              'ex:yetiSecret': "letty's secret",
+            },
+            {
+              '@id': 'ex:yetiPolicy',
+              '@type': ['f:Policy'],
+              'f:targetClass': {
+                '@id': 'ex:Yeti',
+              },
+              'f:allow': [
+                {
+                  '@id': 'ex:globalViewAllowForYetis',
+                  'f:targetRole': {
+                    '@id': 'ex:yetiRole',
+                  },
+                  'f:action': [
+                    {
+                      '@id': 'f:view',
+                    },
+                  ],
+                },
+              ],
+              'f:property': [
+                {
+                  '@id': 'ex:property2',
+                  'f:path': {
+                    '@id': 'ex:yetiSecret',
+                  },
+                  'f:allow': [
+                    {
+                      '@id': 'ex:yetiSecretsRule',
+                      'f:targetRole': {
+                        '@id': 'ex:yetiRole',
+                      },
+                      'f:action': [
+                        {
+                          '@id': 'f:view',
+                        },
+                        {
+                          '@id': 'f:modify',
+                        },
+                      ],
+                      'f:equals': [
+                        {
+                          '@id': 'f:$identity',
+                        },
+                        {
+                          '@id': 'ex:yeti',
+                        },
+                      ],
+                    },
+                  ],
+                },
+              ],
+            },
+            {
+              '@id': did,
+              'ex:yeti': {
+                '@id': 'ex:freddy',
+              },
+              'f:role': {
+                '@id': 'ex:yetiRole',
+              },
+            },
+          ]
+        })
+        .send();
+
+      const signedQuery = client
+        .query({
+          "where": {
+            "@id": "?s",
+            "ex:yetiSecret": "?secret"
+          },
+          "select": "?secret"
+        })
+        .sign();
+
+      let result, error;
+
+      try {
+        result = await signedQuery.send();
+      } catch (e) {
+        console.log(e);
+        error = e;
+      }
+
+      expect(error).toBeUndefined();
+      expect(result).toBeDefined();
+      expect(result).toHaveLength(1);
+      expect(result[0]).toBe("freddy's secret");
```

Comment:
@aaj3f - I ran in the "can't delete from a list" again and so couldn't adequately upsert the policy data for the signed query test case, so I had to create a separate policy and dataset to use in this test case.